config/plugins/visualizations/jqplot/jqplot_bar/src/script.js

```diff
@@ -143,15 +143,15 @@ var CommonWrapper = Backbone.View.extend({
                 xaxis: {
                     label               : chart.settings.get( 'x_axis_label' ),
                     tickOptions: {
-                        angle           : chart.settings.get( '__use_panels' ) === 'true' ? 0 : -30,
-                        showGridline    : chart.settings.get( 'x_axis_grid' ) === 'true'
+                        angle           : this._asBoolean(chart.settings.get( '__use_panels' )) ? 0 : -30,
```

Comment:
Is this the only type change we expect here?  And, can we possibly push this to a layer that doesn't require rebuilding all our charts that use a boolean somewhere?

Reply:
Yes, I think it is the only one, the reason why the `boolean` parameter is so conflicting is that we historically had several definitions of what an actual true-value is i.e. `["true"]`, `"true"`, `True`.  We can now consolidate this with `True`. When the visualization framework is modernized as planned the values will likely be parsed as URL parameters in which case the visualization might need to be able to detect the value as `String` again, thats why it seems to be reasonable to allow visualizations to detect both and not have to worry about it in the future.

Reply:
I reviewed a few more options but came back to the solution here, I think its better to keep the handling of boolean values consistent within Galaxy and let the plugins properly identify parameter values as they fit.

Reply:
@guerler Sounds good, thanks for taking a look.